tests/meta/store/test_cg_meta_store_base.py

```diff
@@ -0,0 +1,103 @@
+"""Test store base module: MIP get files and build bundle"""
+import mock
+import pytest
+
+from snapshottest import Snapshot
+
+import cg.meta.store.base as store_base
+
+from cg.exc import (
+    AnalysisDuplicationError,
+    PipelineUnknownError,
+)
+
+
+@mock.patch("cg.store.Store")
+@mock.patch("housekeeper.store.models")
+def test_add_new_analysis_pipeline_exception(mock_housekeeper_store, mock_status):
+    """
+        test catching the exception when no pipeline is found
+    """
+    mock_bundle = mock_housekeeper_store.Bundle.return_value
```

Comment:
It would be great if you could use the GIVEN/WHEN/THEN syntax for all tests

Reply:
https://github.com/Clinical-Genomics/cg/pull/621/commits/f4d1932a966fae2eaf831da9617630549683b588